addons/account_edi_ubl_cii/wizard/account_move_send.py

```diff
@@ -176,6 +176,9 @@ def _hook_invoice_document_after_pdf_report_render(self, invoice, invoice_data):
         # EXTENDS 'account'
         super()._hook_invoice_document_after_pdf_report_render(invoice, invoice_data)
 
+        if not invoice_data.get('ubl_cii_xml') or not invoice._need_ubl_cii_xml():
```

Comment:
????

Reply:
Got a bug before on runbot tests because it was entering that function, removed it to show you, but it's working fine now, so we can keep it that way